docs/changelog/5_x.rst

```diff
@@ -600,3 +600,80 @@ Bug fixes
 
 * Fix an issue with collection types that affected some migrations.
   (:eql:gh:`#7375`)
+
+5.5
+===
+
+* Fix recompilation slowdowns after several migrations
+  (:eql:gh:`#7099`)
+
+* Fix dumps of types that have both AI and FTS indexes
+  (:eql:gh:`#7405`)
+
+* Fix single-tenant metrics not to filter by tenant
+  (:eql:gh:`#7385`)
+
+* Fix rewrite expressions using ``__specified__`` sometimes
+  generating InvalidReferenceError.
+  (:eql:gh:`#7392`)
+
+* Raise error when using query params in schema.
+  (:eql:gh:`#7400`)
+
+* Add expression to index friendly name.
+  (:eql:gh:`#7401`)
+
+* Treat target and link properties as different when expanding splats.
+  (:eql:gh:`#7402`)
+
+* Remove redundant primary key constraint on 'id'.
+  (:eql:gh:`#7418`)
+
+* workflows: Use an explicit label when selecting runners for builds
+  (:eql:gh:`#7416`)
+
+* Log errors raised when handling ext::ai HTTP requests
+  (:eql:gh:`#7436`)
+
+* ai: Properly forward non-successful responses from non-streaming chat
+  (:eql:gh:`#7440`)
+
+* Fix ai::to_context duplication
+  (:eql:gh:`#7464`)
+
+* Support configuring more arguments with env vars
+  (:eql:gh:`#7470`)
+
+* Use SQL's ``ON CONFLICT`` to implement ``UNLESS CONFLICT`` more often
+  (:eql:gh:`#7472`)
+
+* Fix a race condition where older database configs can overwrite newer ones
+  (:eql:gh:`#7485`)
+
+* Allow subdomains in redirects
```

Comment:
Should we maybe prefix this message with `ext::auth`? Maybe that's worth doing for extension work in general to make it easier for people to know when a server version includes updates to extensions they use.

Reply:
I mentioned `auth` in prose